eos/eos_builder.hpp

```diff
@@ -89,6 +89,24 @@ namespace EOSBuilder {
     }
     return eos;
   }
+
+  template<typename T>
+  EOS applyShiftAndScale(T&& eos,
+                         bool scaled, bool shifted,
+                         Real scale, Real shift) {
+    if (shifted && scaled) {
+      ShiftedEOS<T> a(std::forward<T>(eos),shift);
+      ScaledEOS<ShiftedEOS<T>> b(std::move(a),scale);
+      return b;
+    }
+    if (shifted) {
+      return ShiftedEOS<T>(std::forward<T>(eos),shift);
+    }
+    if (scaled) {
+      return ScaledEOS<T>(std::forward<T>(eos),scale);
+    }
+    return eos;
+  }
```

Comment:
If this is the canonical form used in rage, should we remove support for the other one?

Reply:
I'm not sure it hurts anything to leave it there, but I'm not opposed to removing it either.

Reply:
Well let's leave it for now, I suppose. Because riot does it that way. But we should probably unify that at some point.